tools/anthosbm-ansible-module/roles/gcloud-sdk/tasks/main.yml

```diff
@@ -13,12 +13,6 @@
 # limitations under the License.
 
 ---
-- name: Remove gcloud pacakge source file if present [Ubuntu]
```

Comment:
The reason for removing the package before installation is to keep the role idempotent. Please keep this task.